Dockerfile

```diff
@@ -0,0 +1,8 @@
+FROM golang:latest
```

Comment:
Shall we pin a specific version instead?

Reply:
Also, I'm not sure we should have the build tools in the distributed binaries - perhaps we could use a multi stage build file, like in https://docs.docker.com/language/golang/build-images/#multi-stage-builds